src/euphorie/client/browser/templates/surveys.pt

```diff
@@ -165,7 +165,7 @@
                 <div class="tool-info">
                   <h2 class="tool-title">${tool/title}</h2>
 
-                  <p class="tool-tags tag-cloud">
+                  <p class="tool-tags tag-cloud" tal:condition="nothing">
```

Comment:
@cillianderoiste @ale-rt 
I wonder - this seems to comment out ALL pills: Sector, country and language. The ticket only mentions the sector pills.
The confusion comes maybe from the screen-shot in the ticket, where all 3 pills are marked in red,

Reply:
Whoops! Thanks for keeping an eye on us :hugs:

Reply:
Thanks @pysailor!